src/types/catalog.d.ts

```diff
@@ -75,7 +75,7 @@ export interface AddItemGroupParameters {
 }
 
 export interface AddItemGroupsParameters {
-  itemGroups: ItemGroup[];
+  itemGroups: Omit<ItemGroup, 'children'>[];
```

Comment:
Why do we omit children here?

Reply:
It's only part of the response not the request

Reply:
You're correct it shouldn't be omitted I think I confused it with addItemGroup singular. Fixed!